future-apps/payroll/contracts/Payroll.sol

```diff
@@ -369,6 +369,12 @@ contract Payroll is EtherTokenConstant, IsContract, AragonApp { //, IForwarder {
         emit ChangeAddressByEmployee(employeeId, oldAddress, _newAddress);
     }
 
+    function isForwarder() external pure returns (bool) {
```

Comment:
Can we keep the forwarding functions together, like we do in the other apps (e.g. [Token Manager](https://github.com/aragon/aragon-apps/blob/master/apps/token-manager/contracts/TokenManager.sol#L206))? Even if it breaks the flow a little bit, I think the logical grouping is nice :).